source/_posts/2017-09-01-gsoc-2017-data-visualization-using-daru-view.markdown

```diff
@@ -0,0 +1,284 @@
+---
+layout: post
+title: "GSoC 2017 : Data visualization using daru-view"
+date: 2017-09-01 22:46
+comments: true
+author: Shekhar Prasad Rajak
+categories: [GSoC,GSoC 2017,Students,Data Visualization,daru,daru-view,
+  Charts,Plotting,Data Analysis]
+---
+
+Hello, Rubyist! Are looking for good ruby gem for interactive data
+visualisation? Then you must try [`daru-view`](https://github.com/Shekharrajak/daru-view), a plugin gem for [`daru`](https://github.com/SciRuby/daru).
+
+## What makes daru-view different ?
+
+* `daru-view` is desinged for interactive plotting of charts and tables.It
+provide different plotting tools like Nyaplot, HighCharts, GoogleCharts,
+DataTable. So you don't have to write any JavaScript code from these sites
+and no need to shift to other language to get charts.
+
+* It can work with **any ruby web application framework like Rails/Sinatra/Nanoc**. If you want to try few examples then please look into the
+`daru-view/spec/dummy_*` examples of Rails, Sinatra, Nanoc web applications.
+
+* Now Ruby developers are using IRuby notebook for few lines of code.
```

Comment:
Rephrase this as `Now Ruby developers are using IRuby notebook for interactive programming`.